apps/organization/src/organization.service.ts

```diff
@@ -250,7 +250,7 @@ export class OrganizationService {
 
       const organizationExist = await this.organizationRepository.checkOrganizationExist(updateOrgDto.name, orgId);
```

Comment:
you can optimize this code
1. get org by name
2. check if org.Id received is same as orgId, if true, throw new ConflictException(ResponseMessages.organisation.error.exists);

Reply:
Resolved